wire-ios-data-model/Source/Model/Message/ConversationMessage.swift

```diff
@@ -293,7 +293,7 @@ extension ZMMessage: ZMConversationMessage {
 
     public var isRestricted: Bool {
         guard
-            (self.isFile || self.isImage),
+            self.isFile || self.isImage,
```

Comment:
`self.isFile` should be good enough. We could skip `self.isImage`

Reply:
let's explore that on a separate PR. On this one, I would like to focus on fixing the warnings.